Telegram/SourceFiles/mtproto/rsa_public_key.cpp

```diff
@@ -30,7 +30,7 @@ using std::string;
 
 namespace MTP {
 namespace {
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Shouldn't we compare `LIBRESSL_VERSION_NUMBER` with some number, too?

Reply:
OpenSSL wiki seems to recommend [the same approach](https://wiki.openssl.org/index.php/Talk:OpenSSL_1.1.0_Changes):
```cpp
#if (OPENSSL_VERSION_NUMBER < 0x10100000L) || defined (LIBRESSL_VERSION_NUMBER)
```

So I think that's okay.